packages/dashmate/configs/system/base.js

```diff
@@ -87,12 +87,7 @@ module.exports = {
         docker: {
           image: 'dashpay/envoy:0.24-dev',
         },
-        http: {
```

Comment:
Shall we keep `http.port` in case if need one more in future? I.e. admin console?